Jenkinsfile

```diff
@@ -211,21 +213,19 @@ pipeline {
           if (isTriggered()) {
             buildInfo += """
             Triggered by: ${upstreamJobName} #${upstreamBuildNumber}
-            """
-            if (isTriggeredByNuxeoPR()) {
-              buildInfo += """
             With parameter NUXEO_VERSION: ${params.NUXEO_VERSION}
-              """
-            }
+            """
           } else {
             buildInfo += """
             Not triggered by an upstream job.
             """
           }
-          buildInfo += """
+          if (!isTriggered() || isTriggeredByNuxeoPR()) {
```

Comment:
Maybe worth extracting another function for all the `!isTriggered() || isTriggeredByNuxeoPR()` with a meaningful name when reading the pipeline script?

Maybe this `if` could also be added to the `setGitHubBuildStatus`?

Reply:
> Maybe worth extracting another function for all the !isTriggered() || isTriggeredByNuxeoPR() with a meaningful name when reading the pipeline script?

Yes!

> Maybe this if could also be added to the setGitHubBuildStatus?

I'd rather not since the clause is different depending on the status, for `yarn/eslint` we only want to set the status`if (!isTriggered()) {`.